CHANGELOG.rst

```diff
@@ -66,6 +66,7 @@ Interactive improvements
 - The :kbd:`Alt-H` binding will now show the manpage of the command under cursor instead of the always skipping ``sudo`` and the likes (:issue:`9020`).
 - If ``$fish_color_valid_path`` contains an actual color instead of just modifiers, those will be used for valid paths even if the underlying color isn't "normal" (:issue:`9159`).
 - Performance improvements to highlighting (:issue:`9180`) and the cd completions (:issue:`9220`) should make using fish more pleasant on slow systems.
+- When running fish inside SSH, :kbd:`Control-X` now copies to the local system's clipboard if the terminal supports OSC 52. 
```

Comment:
I would change "local system's" to "SSH client's" because it could be confusing what is local and what is remote.